README.md

```diff
@@ -118,6 +119,13 @@ package declarations found in merged files will overwrite the declarations
 made by earlier files. Files are loaded in the order specified by the
 `include` setting with globbed files being processed in alphabetical order.
 
+### skip
+
+By default, Composer's conflict resolution engine is used to determine which
+version of a package should be installed when multiple files specify the same
+package. A `"skip": true` setting can be provided to change to a "skip" 
+resolution strategy. In this mode, duplicate package declarations found in 
+merged files will be ignored, and the root file will take precedence.
```

Comment:
> ### skip
>
> By default, Composer's conflict resolution engine is used to determine which
version of a package should be installed when multiple files specify the same
package. A `"skip": true` setting can be provided to change to a "skip" 
resolution strategy. In this mode, duplicate package declarations found in 
merged files will be ignored, and the root file will take precedence.

Change to:

### ignore-duplicates

By default, Composer's conflict resolution engine is used to determine which version of a package should be installed when multiple files specify the same package. An `"ignore-duplicates": true` setting can be provided to change to a "first version specified wins" conflict resolution strategy. In this mode, duplicate package declarations found in merged files will be ignored in favor of the declarations made by earlier files. Files are loaded in the order specified by the `include` setting with globbed files being processed in alphabetical order.

Note: `"replace": true` and `"ignore-duplicates": true` modes are mutually exclusive. If both are set, `"ignore-duplicates": true` will be used.